nilearn/decoding/tests/test_searchlight.py

```diff
@@ -175,3 +176,44 @@ def test_searchlight_group_cross_validation_with_extra_group_variable(
     # run searchlight on list of 3D images
     sl = searchlight.SearchLight(mask_img)
     sl.fit(imgs, y)
+
+
+def test_searchlight_attributes_exist_after_fit():
+    """Test if attributes `process_mask_` and `masked_scores_`
+    exist after fitting using mock data.
+    """
+    # Create mock data instead of using fetch_haxby()
+    data = np.random.default_rng().random(
```

Comment:
I think you can make your life easier here by reusing the function `_make_searchlight_test_data` defined earlier in this module.

It will also provide you with a mask_img.